packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -34,10 +35,16 @@ export class CommandRegistryImpl implements CommandRegistryExt {
     private readonly commands = new Set<string>();
     private readonly handlers = new Map<string, Handler>();
     private readonly argumentProcessors: ArgumentProcessor[];
+    private readonly commandsConverter: CommandsConverter;
```

Comment:
CommandRegistryImpl never does anything with the "commandsConverter". IMO, the converter should be owned by 
"TreeViewExtImpl", not the command registry.

Reply:
This can be used by other APIs.